vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -22097,6 +22097,16 @@ unit:TONNE-PER-HA
   rdfs:label "tonne per hectare"@en ;
   owl:sameAs unit:TON_Metric-PER-HA ;
 .
+unit:TONNE-PER-HA-YR
+  a qudt:DerivedUnit ;
+  a qudt:Unit ;
+  qudt:conversionMultiplier 0.1 ;
```

Comment:
Things are much improved! I have now taken a look at the conversionMultiplier. Please check my math, but I think the correct value should be approximately 3.17 x 10^-9, or 0.00000000317
This is the number needed to convert to KiloGM-PER-M2-SEC

Reply:
Hi Steve - I have amended the conversionMultiplier to read 0.00000000317. (your math is correct)

Thanks again